### PR TITLE
Search implemented

### PR DESCRIPTION
**Pull request by AlexanderSandelius**
First implementation of the Search class which was required in the laboratory assignment.

### CODE REVIEW

File: Search.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+                    System.out.println(line);
+                }
+            }
+        } catch (FileNotFoundException e) {
+            System.out.println(e.toString());
+        }
+    }
+
+    public static void main(String[] args) {
+        readFile(args[1], args[2]);
```
Comment (johanhummel) [inline Search.java]: Consider printing informative message if missing command line arguments

Comment (carlofthetower) [inline Search.java]: Consider handing for null input

**Review by carlofthetower — CHANGES_REQUESTED** [review]
Look at comment

**Review by johanhummel — APPROVED** [review]
LGTM

**Review by carlofthetower — APPROVED** [review]
**Cool**

**Pull request merged**